packages/ketchup/src/components/kup-echart/kup-echart.tsx

```diff
@@ -437,6 +440,74 @@ export class KupEchart {
         } as echarts.EChartsOption;
     }
 
+    #bubbleChart() {
+        const x = this.#createX(),
+         y = this.#createY(),
+         data = [], temp=[], legend={}, series=[];
+        
+        for(let i =0; i<x.length; i++){
+
+            temp.push([y['Gdp'][i], y['Age'][i], y['Size'][i], y['Country'][i], x[i] ]);
+        }
+
+        const year = [... new Set(x)];
```

Comment:
As explained above, "years" might not be present as columns are dynamic. A variable shouldn't be named to reflect sample data.